BackwardElimination.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import statsmodels.api as sm
+import numpy as np
+
+
+def backward_elimination(x, y, sl):
+    x = pd.DataFrame(np.append(np.ones((x.shape[0], 1)).astype(int), x, axis=1))
+    rSqData = pd.DataFrame(r_sq_elimination(x, y))
+    finalModel = p_elimination(rSqData, y, sl)
+    return finalModel
+
+
+def r_sq_elimination(x, y):
+    x = pd.DataFrame(x)
+    ols_regeressor = sm.OLS(exog=x, endog=y).fit()
+    bestValue = float("{0:.4f}".format(ols_regeressor.rsquared_adj))
+    noOfColumn = x.shape[1]
+    bestModel = pd.DataFrame(x)
+    foundNew = False
+    for i in range(1, noOfColumn):
+        temp = x.drop(x.columns[i], axis=1)
```

Comment:
You could loop through column names like

```
for col in x.columns:
    temp = x.drop(col, axis=1)
```

Reply:
Done.

Reply:
You seem to have reverted back?

Reply:
Yes. I ran and realised that something was going wrong when I changed it. And secondly I do not want to work on the 0th column. So this is saving me resources.